api/hypertable/create_table.md

```diff
@@ -9,6 +9,9 @@ api:
 products: [cloud, mst, self_hosted]
 ---
 
```

Comment:
I think this page is becoming unmanageable with all the information we're putting in it. Would be good to rethink it in migration.

Reply:
ok